metadata/src/utils/validation.rs

```diff
@@ -1236,9 +1227,15 @@ mod tests {
             m
         };
 
+        let name_set: Vec<String> = trimmed_metadata
+            .pallets
+            .values()
+            .iter()
+            .map(|pallet| pallet.name.to_string())
+            .collect();
         // test that the hashes are the same:
         let hash = MetadataHasher::new(&metadata)
-            .only_these_pallets(&["First"])
+            .only_these_pallets(&name_set)
```

Comment:
I guess we can revert this to "First" now to simplify?

Reply:
removed the test completetely as the behaviour in `OuterEnumHashes` relied on the fact that some of the variants in outernums are removed unconditionally.